### ISSUE TITLE
Cannot remove blue

### ISSUE THREAD

**Issue by cdecinkoKnight** [alexandermendes/vue-confetti#39]
No matter what colors you specify, default colors or colors for each particle, you always get blue in addition to your selected colors.

**Comment by mmikkel** [comment]
Same here. I'm trying to create confetti in 3 different shades of pink only, but the blue confetti persists.

Here's my config:

```js
this.$confetti.start({
    canvasId,
    particles: [{
        type: 'rect',
        colors: ['#FFF3F6', '#FDE2E9', '#F4B3CE']
    }]
});
```

I've also tried setting to set the `defaultColors` array to the same colors, but same result.

**Comment by alexandermendes** [comment]
Ah, thanks, just fixed with v2.0.7!
